jms/org.wso2.carbon.transport.jms/src/main/java/org/wso2/carbon/transport/jms/factory/JMSConnectionFactory.java

```diff
@@ -571,9 +571,9 @@ private Destination createDestination(Session session, String destinationName) t
              */
             try {
                 if (this.destinationType.equals(JMSConstants.JMSDestinationType.QUEUE)) {
```

Comment:
Also @Megala21 and @amjadhifthikar , I see few IDE warnings with this class, can we fix them all please?